lighthouse-core/audits/time-to-interactive.js

```diff
@@ -71,29 +71,29 @@ class TTIMetric extends Audit {
       if (fmpResult.rawValue === -1) {
         return generateError(fmpResult.debugString);
       }
-      const fmpTiming = parseFloat(fmpResult.rawValue);
-      const timings = fmpResult.extendedInfo && fmpResult.extendedInfo.value &&
-          fmpResult.extendedInfo.value.timings;
 
       // Process the trace
       const tracingProcessor = new TracingProcessor();
       const trace = artifacts.traces[Audit.DEFAULT_PASS];
       const model = tracingProcessor.init(trace);
       const endOfTraceTime = model.bounds.max;
 
-      // TODO: Wait for DOMContentLoadedEndEvent
-      const fMPts = timings.fMP + timings.navStart;
+      const fmpTiming = fmpResult.rawValue;
```

Comment:
good question. I'm gonna normalize all the exported rawValues to be `parseFloat(num.toFixed(1))` so we never have to parseFloat randomly.